js/interface.js

```diff
@@ -667,10 +904,21 @@ $('.file-manager-wrapper')
         options.appId = lastFolderSelected.appId;
       }
     }
-
-    Fliplet.Media.Folders.create(options).then(addFolder);
-
-    $('.new-btn').click();
+    console.log('lastFolderSelected', lastFolderSelected)
+    Fliplet.Media.Folders.create(options).then( function (folder) {
+      if (!isCreatingInModal) {
```

Comment:
looks like these two IFs should be combined into a single block

Reply:
fixed